l10n_it_riba/tests/test_riba.py

```diff
@@ -65,6 +65,18 @@ def test_delete_due_cost_line(self):
         # Collection fees line has been unlink
         self.assertEqual(len(self.invoice.invoice_line_ids), 1)
 
+    def test_date_maturity_invoice_line(self):
```

Comment:
Questo test passa anche senza la fix al modulo fatta in questa PR, invece dovrebbe fallire senza la fix e passare quando la fix è presente.
Puoi fare in modo che almeno un test fallisca senza la fix?
Altrimenti non c'è modo di sapere se qualche altra modifica causerà una regressione.

Reply:
@Borruso puoi fare la modifica?